lib/membership/member.js

```diff
@@ -242,4 +217,26 @@ Member.isStatusPingable = function isStatusPingable(status) {
     }
 };
 
+/**
+ * Get the precedence of a status.
+ *
+ * @param {string} status a valid status (@see {Member.Status})
+ * @returns {number} A higher number would mean that the status is considered as a higher priority.
+ */
+Member.statusPrecedence = function statusPrecedence(status) {
+    switch (status) {
+        case Member.Status.alive:
+            return 0;
+        case Member.Status.suspect:
+            return 1;
+        case Member.Status.faulty:
+            return 2;
+        case Member.Status.leave:
+            return 3;
+        default:
+            // unknown states will never have precedence
+            return -1;
```

Comment:
Motiejus is right: the higher the number, the higher the priority. A minus one means the opposite: lowest priority,.